src/test/java/core/basesyntax/service/impl/DataProcessorServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.FruitShopException;
+import core.basesyntax.service.DataProcessorService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataProcessorServiceImplTest {
+    private static List<String[]> list;
+    private DataProcessorService dataProcessorService = new DataProcessorServiceImpl();

```

Comment:
such fields should be initialized in methods with @BeforeClass, the same for other test classes